website/docs/api/cli.mdx

```diff
@@ -249,6 +250,94 @@ $ python -m spacy init labels [config_path] [output_path] [--code] [--verbose] [
 | overrides         | Config parameters to override. Should be options starting with `--` that correspond to the config section and value to override, e.g. `--paths.train ./train.spacy`. ~~Any (option/flag)~~                         |
 | **CREATES**       | The label files.                                                                                                                                                                                                   |
 
+## configure {id="configure", version="TODO"}
+
+Modify or combine existing configs in high-level ways. Can be used to automate
+config changes made as part of the development cycle.
```

Comment:
Could you elaborate on this? "high-level ways" and "changes made as part of the development cycle" doesn't seem very informative to me.

Reply:
It'd be great to have any suggestions here, I had trouble summarizing the functionality more specifically since it really is a variety of high-level config modifications. 

The reference to the development cycle is about starting with small models to iron out data issues and switching to larger models (like transformers) only later. Similarly, it's common for a user to train a single specific model, like a textcat, and then want to combine it with one of the pretrained pipelines for components that they want but don't need to re-train, like a lemmatizer.

Reply:
I see what you mean. Maybe it's easier to describe it a less abstract manner? Feel free to adjust in any way.

> Modify or combine existing configs. Can be used for e. g. swapping feature sources, creating configs to resume training or merging several configs into a single one. This may simplify parts of the development cycle. For example, it allows starting off with a config for a faster, less accurate model (using the CNN tok2vec) and conveniently switching to transformers later, without having to manually adjust the config.